Eastern-Research-Group/expert-query
# Removed unused OpenAPI spec and API route

## Related Issues:
* [EQ-348](https://jira.epa.gov/browse/EQ-348)
* [EQ-349](https://jira.epa.gov/browse/EQ-349)

## Main Changes:
* Removed the `/api/getFile` endpoint from the server and OpenAPI spec, since it is not being used in the app.
* Fixed a typo in the OpenAPI spec.

## Steps To Test:
1. Go to http://localhost:3000/api-documentation.
2. Attempt to query some of the routes with the "Try it out" tool, and make sure nothing is broken.
3. Test the OpenAPI specifications with the OWASP ZAP tool, if possible.
